features/step_definitions/column_steps.rb

```diff
@@ -0,0 +1,8 @@
+Then /^I should see a columns container$/ do
+  expect(page).to have_css '.columns'
+end
+
+Then /^I should see (a|\d+) columns?$/ do |count|
+  count     = count == 'a' ? 1 : count.to_i
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.